redux_userlist_06/src/App.tsx

```diff
@@ -1,21 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
+import { useEffect } from "react";
 
-import { Link, Outlet } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import axios from "axios";
+
+import { addUserList } from "../src/actions/actions";
+import { UserDetails } from "./interfaces";
+import withLoading from "./components/Dashboard/withLoading";
```

Comment:
import sequence should be like 
1. third party modules
2. custokm components
3. helper files
4. constants files
6. css/ image/ icons file
there has to be one blank line betwwen these sections